server/controllers/customers/csv.customers.js

```diff
@@ -0,0 +1,68 @@
+const model = require('../../models')
+
+const fs = require('fs')
+const csv = require('fast-csv')
+
+const sanitizeEmails = require('../../public/javascript/sanitize')
+
+
+// Upload and save CSV file to database
+const upload = async (req, res) => {
+  try {
```

Comment:
Aha! don't worry I used to make this mistake too. You guys don't have to refactor this, but controllers shouldn't have too much logic. Check this article [explaining why](https://medium.com/swlh/your-controllers-are-doing-too-much-this-is-how-to-simplify-them-7f7d0ea0a810)
This csv file handler could have been a utils file that its main goal is handle CSV files. A **very important** topic, single responsibility! Here is a cool [article too](https://medium.com/@learnstuff.io/single-responsibility-principle-ad3ae3e264bb)